controllers/ManagementController.php

```diff
@@ -330,5 +330,67 @@ private function createUserToken()
     {
         return Util::getRandomString(32);
     }
+
+    /**
+     * @param null $id
+     * @throws CannotCreateTokenException
+     * @throws CannotSaveException
+     */
+    public function actionResetPassword($id = null)
+    {
+        $user = $this->checkIfUserExist($id);
+
+        $this->resetPassword($user);
+    }
+
+    /**
+     * @param $user
+     * @throws CannotCreateTokenException
+     * @throws CannotSaveException
+     * @throws \Exception
+     */
+    private function resetPassword($user)
+    {
+        $user->password_hash = null;
```

Comment:
Don't forget to set `$user->is_active` to `false`, otherwise it will cause a fatal error when the user tries to log in without re-creating his password.